open_academy/views/open_academy_course_views.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="open_academy_course_action" model="ir.actions.act_window">
+        <field name="name">Courses</field>
+        <field name="res_model">open.academy.course</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="open_academy_course_tree" model="ir.ui.view">
+        <field name="model">open.academy.course</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" string="Courses"/>
```

Comment:
The attribute "string" is not necessary here, because in the model you already added the string parameter.

The string is used only when we want the label visible to the user to be different from the Odoo default (field's name).

If we add it in the model, it is no longer necessary to add it in the view and vice versa.